### DISCUSSION TITLE
Can't proccess my application while my profile is complete

### DISCUSSION THREAD

**Discussion by GmrezaH** [community/community#67980] [GitHub Education]
### Hi everyone,

I provided Document from my college that says I'm studying there and my Student ID card in proof of academic status but it won't let me "Proccess my Application" and it says complete your github user profile while my profile is complete based on the github documentation and even I converted my profile name in my language (because my college couldn't provide me with translationed document but I mentioned my Student ID and validity in the image I provided for proof). 
Can anyone help me please?
![photo1695406385](https://github.com/community/community/assets/99497495/11945e10-c909-4ccf-9d52-1b7c5e60d18c)

**Comment by SumanthSShet** [comment]
If you are getting the error "Can't proccess my application while my profile is complete" when applying for GitHub student benefits, even though your profile is complete, there are a few things you can try:

Make sure that your profile information is accurate and up-to-date.
Make sure that your profile picture is a clear and recognizable photo of yourself.
Make sure that your profile bio includes a brief description of yourself and your academic status.
Make sure that you have uploaded a valid proof of academic status.
If you have done all of the above and you are still getting the error, you can try the following:

Contact GitHub support for help.
Try applying for GitHub student benefits again from a different browser.
Try applying for GitHub student benefits again from a different device.
If you are posting in the GitHub community forum, be sure to include the following information:

The error message you are getting
The steps you have already taken to try to fix the problem
A link to your GitHub profile
This will help the community members to troubleshoot the issue and provide you with the best possible assistance.

Here are some additional tips:

If your college couldn't provide you with a translated document, you can try to get a translation from a trusted source, such as a professional translation service.
You can also try to get a letter from your college on official letterhead that confirms your academic status.
If you are still having trouble, you can contact GitHub support and explain your situation. They may be able to help you manually process your application.
